### DISCUSSION TITLE
Video Downloadhelper for Mac Pro 2011

### DISCUSSION THREAD

**Discussion by Bobby3469** [aclap-dev/video-downloadhelper#487] [Help and Q&A]
### Your question

Good morning, I have problems installing the integrative external application, the software vdhcoapp-mac-x86_64-installer.
I own a 2011 MacBook Pro - macos High Sierra.
Can you help me? How can I?

### Addon & CoApp version and details

If the app is not recognized:

The browser you're using (be as specific as possible)
OS (Linux/Windows/Mac) & OS version
OS 64bits or 32bits
Link to the coapp you installed
On Mac, the output of /Applications/net.downloadhelper.coapp.app/Contents/MacOS/vdhcoapp --info
On Windows, open a CMD shell, go to the install directory, and type vdhcoapp.exe --info
On Linux, your sandbox mechanism (if any): flatpak/snap/…
On Linux, the output of /opt/vdhcoapp/vdhcoapp --info (or ~/.local/share/vdhcoapp/vdhcoapp --info if it was install with the script)
If the download video is not valid or missing audio:

version of Video DownloadHelper addon
link to the page with the video

**Comment by jcfrog** [comment]
I'm afraid Sierra is a bit too old for the CoApp: https://github.com/aclap-dev/video-downloadhelper/wiki/CoApp-Installation#download-for-macos-1015-and-above
